reference/changelog.md

```diff
@@ -71,6 +76,95 @@ redirect_from: /install/changelog.html
+
+- Many improvements for Go development.  Thanks to [@justone](https://github.com/justone)!. [pulumi/pulumi#1954](https://github.com/pulumi/pulumi/pull/1954) [pulumi/pulumi#1955](https://github.com/pulumi/pulumi/pull/1955) [pulumi/pulumi#1965](https://github.com/pulumi/pulumi/pull/1965)
+- Extend `pulumi.output` to deeply unwrap `Input`s.  This significantly simplifies working with `Inputs` when building Pulumi components.  [pulumi/pulumi#1915](https://github.com/pulumi/pulumi/pull/1915)
+
+### @pulumi/aws 0.16.0
+
+- Moved all `@pulumi/aws-serverless` features down into `@pulumi/aws`.  Event handlers are now available as methods like `bucket.onObjectCreated` off of instances of `aws.s3.Bucket` and all other Lambda event sources.  In addition, the `aws.apigateway.x.API` type is available for simple construction of API Gateways directly in the `@pulumi/aws` package.  
+- Added richer typing for many EC2 and Autoscaling resources.  Thanks to [@jen20](https://github.com/jen20)! [pulumi/pulumi-aws#323](https://github.com/pulumi/pulumi-aws/pull/323) [pulumi/pulumi-aws#324](https://github.com/pulumi/pulumi-aws/pull/324)
+- Adopted `v1.38.0` of the AWS Terraform Provider. [pulumi/pulumi-aws#341](https://github.com/pulumi/pulumi-aws/pull/341)
+
+### @pulumi/aws-infra 0.16.0
+
+- Updated to latest `@pulumi/pulumi` and `@pulumi/aws` dependencies.
+
+### @pulumi/aws-serverless 0.16.0
+
+- This library is now deprecated.  All features of this library are now avaiable directly in `@pulumi/aws`.
+
+### @pulumi/azure 0.16.0
+
+- Adopted `v1.15.0` of the AzureRM Terraform Provider. [pulumi/pulumi-azure#125](https://github.com/pulumi/pulumi-azure/pull/125)
+- Many iprovements to autoname generation to match Azure resource name requirements.
+- **[Breaking Change]** Fix name of `azure.role.Assignment` from previous `azure.role.assignment`. [pulumi/pulumi-azure#127](https://github.com/pulumi/pulumi-azure/pull/127)
+
+
+### @pulumi/cloud, @pulumi/cloud-aws, @pulumi/cloud-azure 0.16.0
+
+- Implementations of many `@pulumi/cloud-aws` APIs moved over to build on top of higher-level APIs in `@pulumi/aws`.  This may cause some resources, such as `aws.lambda.Function` and `aws.apigateway.RestAPI` to be recreated on updates from previous versions.
+
+### @pulumi/docker
+
+- Pass logging messages to the right sink when doing docker operations. [pulumi/pulumi-docker#7](https://github.com/pulumi/pulumi-docker/pull/7)
+- Add `docker.Image` component. [pulumi/pulumi-docker#10](https://github.com/pulumi/pulumi-docker/pull/10)
+
+### @pulumi/gcp 0.16.0
+
+- Adopted `v1.15.0` of the Google Terraform Provider. [pulumi/pulumi-gcp#51](https://github.com/pulumi/pulumi-gcp/pull/51)
+
+### @pulumi/kubernetes 0.17.0
+
+TODO: @hausdroff
+
+### @pulumi/vsphere 0.15.0
+
+Added a new provider for VMware vSphere, enabling deployment and management of resources in vCenter Server and ESXi. Thanks [@Smithx10](https://github.com/Smithx10)!
+
+### @pulumi/vsphere 0.15.0
```

Comment:
Should we rebuild OpenStack, vSphere and eks with the 0.16.0 Pulumi references?

Reply:
Yes - let's go ahead and do this for consistency.